asciimatics/widgets/frame.py

```diff
@@ -140,14 +212,8 @@ def fix(self):
         fill_height = y = 0
         for _ in range(2):
             # Pick starting point/height - varies for borders.
-            if self._has_border:
```

Comment:
I suspect the Frame docstrings need updating to reflect the restriction that has been removed.